docs/node-operators/how-to/rotate-vega-keys.md

```diff
@@ -8,102 +8,233 @@ hide_title: false
-`TARGET_HEIGHT` - a block height when the key rotation will take place. It has to be in the future. It is recommended to do it at least 30-50 blocks in future because there will be more time to do step 4 in the meantime.
-`BASE64_TRANSACTION` - transaction output by step `3.A`
-`VEGA_NETWORK` - the Vega network that the validator node is running on
+```
+✓ Key rotation succeeded
 
-**3.A** This step will create a key rotation submission transaction and output it as a Base64 string. The output transaction is referred to as `BASE64_TRANSACTION`
+Transaction (base64-encoded):
+CpoBCLvli5S+lOaH9gEQZKp9iQEIChCWARpAYmJmNTA3OWI4MDBhOTNjN2U5Nzc1NDdmNDVhN2QwMzUzYWEyZTUyYjBjNzQ1ZmYwOT
+BmYzc5NWQ5MDEyYTYwNCJAZDRjYzEyMjI1ZDUyZDExMDVkZmI2ZDcyZjJhODQyYmUwNGM3OTUzYTY2OGYxYTllNTQyODVhM2Q2MzU5
+ODJlNhKTAQqAATlmYTk4OGE0MDdmZGU0Yjk4Y2E0ZTg0NjRlYTExZWViYjhkOTIzNGM3YzRhNWNlOTJlYzM2ODEwOGEzYzcyYzZmNj
+IzNmIxMWZmNDQ0NDMzMzk5MTNmZjA0Y2I0MTU3NTRjYTY2Yzg5YzUyYjQ3YzE1ZjYxYjk0ODMzZjUxMjBiEgx2ZWdhL2VkMjU1MTkY
+AYB9A9I+QDQ0ZGUxNzc0Y2ViOWJmYjVlOWIzYzk0ZTE0NTMyMjRjYTA4NzA0ZjkxYzIyYWQxMTJlMDdmYWNjNGVmNTMyZDg=
 
+Master public key used:
+44de1774ceb9bfb5e9b3c94e1453224ca08704f91c22ad112e07facc4ef532d8
 ```
-vegawallet --home "WALLET_PATH" key rotate --current-pubkey "CURRENT_PUB_KEY" --new-pubkey "GENERATED_PUB_KEY" --tx-height "TX_HEIGHT" —target-height "TARGET_HEIGHT" --wallet "WALLET_NAME"
+
+The transaction will be signed by the wallet's master key which is necessary for the network to accept the rotation request. The base64-encoded transaction string should be noted down or saved to a file.
+
+
+:::note 
+It is advised to choose `TARGET_HEIGHT` to be away from an epoch boundary to allow enough time to transfer self-delegations between the rotated Vega keys
+:::
+
+## Transfering the Artefacts
+
+The isolated wallet file and the encoded transaction will need to be transported to the machine that is running the Validator node. This is so that the isolated wallet file can be imported and the key rotation transaction can be sent to the network.
+
+How this is done is dependent on the hardware and networking available to you.
+
+
+## Submitting the Key Rotation Transaction and Updating the Node Wallet
+The following steps should be executed on the machine running the Validator node. The artefacts generated in the previous steps should be in an accessible location.
+
+### Send the Vega key rotation transaction to the network
+
+The pre-signed `ENCODED_TRANSACTION` can be sent to the network via the running Validator node using the following command:
+
 ```
+vega wallet tx send ----node-address NODE_ADDRESS BASE64_TRANSACTION
+```
+
+where `NODE_ADDRESS` is the address of the gRPC endpoint for the running Validator node. This will stage the key rotation to take place at the previously chosen block height.
```

Comment:
```suggestion
where `NODE_ADDRESS` is the address of the gRPC endpoint for the running validator node. This will stage the key rotation to take place at the previously chosen block height.
```